### PR TITLE
rose stem without the hyphen

### PR DESCRIPTION
**Pull request by wxtim**
Following https://github.com/metomi/rose/pull/2494 it will be possible to register `stem` as a `rose` command.

### CODE REVIEW

**Comment by oliver-sanders** [comment]
(Note should `rm $(which rose-stem)` as pip will not remove this script when you `pip install` from this branch)

**Review by oliver-sanders — APPROVED** [review]

File: cylc/rose/stem.py [hunk 1]
```diff
@@ -17,11 +17,7 @@
 # You should have received a copy of the GNU General Public License
 # along with Rose. If not, see <http://www.gnu.org/licenses/>.
 # -----------------------------------------------------------------------------
-"""Cylc 8 implementation of 'rose stem'
-
-Provide a wrapper around "cylc install" for rose-stem suites.
-
-To run a rose-stem suite use "cylc play".
```
Comment (oliver-sanders) [inline cylc/rose/stem.py:24]: This line would be useful to keep (old `rose-stem` users will get confused when nothing happens).

**Review by oliver-sanders — APPROVED** [review]

**Comment by oliver-sanders** [comment]
@datamel, could you give this a quick glance over when you get the chance.

Note: tests will be broken until the Python CLI stuff in Rose is merged.

**Comment by oliver-sanders** [comment]
Good to go (@wxtim please squash commits).

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -40,6 +41,39 @@ jobs:
...
+            `lsb_release -cs` svn19" \
+            >> /etc/apt/sources.list.d/subversion19.list'
+          sudo wget -q http://opensource.wandisco.com/wandisco-debian.gpg -O- \
+            | sudo apt-key add -
+
+          # prepend FCM bin to $PATH
+          FCM_PATH="$GITHUB_WORKSPACE/fcm/bin"
+          # the github actions way (needed for cylc jobs)
+          echo "$FCM_PATH" >> "${GITHUB_PATH}"
+          # the bashrc wat (needed for subsequent gh action steps)
```
Comment (datamel) [inline .github/workflows/tests.yml:74]:
```suggestion
          # the bashrc way (needed for subsequent gh action steps)
```
typo here?

**Review by datamel — APPROVED** [review]
Thanks @wxtim. No problems found for me (only minor typo in comment)

**Pull request merged**